### ISSUE TITLE
Error: JSON-RPC error 12000: itch.io API error (400): invalid game

### ISSUE THREAD

**Issue by rhys-vdw** [itchio/itch#2203]
I'm getting this error on my game page in the app. It's appearing where I'd expect the "Download" link to be. Currently the game is password protected, but I didn't find anything that suggests that this would be a problem.

![image](https://user-images.githubusercontent.com/822105/48248000-9b0f0b00-e449-11e8-8d7e-14094a98f1f9.png)

I uploaded the game builds with Butler, and they can be downloaded and executed fine via the web interface.

So I see three problems here:

- There is a problem which is causing this error.
- Error message is confusing and uninformative. I don't really understand what's going on here, but the real error seems to be "invalid game" from the Itch API.
- It is not possible to copy the error text, or the stack trace in the tooltip (as soon as the mouse moves it disappears). This makes reporting errors like this a bit cumbersome.

Let me know if I can provide any more information. I was hoping we could set up our game to download through the app tomorrow.

**Comment by rhys-vdw** [comment]
I found info that games restricted with password are not distributable via the app [here](https://itch.io/post/94903). I think this could be communicated more clearly if it is indeed the problem.

**Comment by fasterthanlime** [comment]
I agree, this format of error shouldn't appear unless something fundamental is wrong. (And even then, it should probably be clearer / more actionable).

I'm curious though: did you see this for your own game *and* logged in as the account that uploaded it? Because no matter whether the game is in draft, restricted, or public visibility, the creator should always be able to see it.

**Comment by rhys-vdw** [comment]
I was logged in to my personal account, the game was in our company account.

**Comment by tsjost** [comment]
I was hit by this bug today trying to give players access to certain files through a Download Key. Opening the URL in the desktop app displays the "API: invalid game" error instead of the Download button.

Probably related to #1080. The desktop app seems to be reading data fine from the URL/embedded page, so how come it can't just pass the key/password through to the /games API?
